src/frontend-samples/view-clip-sample/ViewClipApp.tsx

```diff
@@ -0,0 +1,96 @@
+/*---------------------------------------------------------------------------------------------
+* Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+* See LICENSE.md in the project root for license terms and full copyright notice.
+*--------------------------------------------------------------------------------------------*/
+import * as React from "react";
+import "@bentley/icons-generic-webfont/dist/bentley-icons-generic-webfont.css";
+import "../../common/samples-common.scss";
+import ViewClipUI from "./ViewClipUI";
+import { EditManipulator, IModelApp, IModelConnection, ScreenViewport, ViewClipClearTool, ViewClipDecorationProvider, Viewport } from "@bentley/imodeljs-frontend";
+import { ClipMaskXYZRangePlanes, ClipPlane, ClipPrimitive, ClipShape, ClipVector, ConvexClipPlaneSet, Plane3dByOriginAndUnitNormal, Point3d, Vector3d } from "@bentley/geometry-core";
+
+export default class ViewClipApp {
+  public static async setup(iModelName: string, iModelSelector: React.ReactNode) {
+    return <ViewClipUI iModelName={iModelName} iModelSelector={iModelSelector} />;
+  }
+}
+
+/* Method for clearing all clips in the viewport */
+export function clearClips(vp: ScreenViewport) {
```

Comment:
Why export a bunch of separate functions?  The pattern from the other samples would be to put all of these into the App class which is already exported.